Doc/library/sqlite3.rst

```diff
@@ -21,17 +21,17 @@ The sqlite3 module was written by Gerhard Häring.  It provides a SQL interface
 compliant with the DB-API 2.0 specification described by :pep:`249`, and
 requires SQLite 3.7.15 or newer.
 
-To use the module, you must first create a :class:`Connection` object that
+To use the module, start by creating a :class:`Connection` object that
 represents the database.  Here the data will be stored in the
 :file:`example.db` file::
 
    import sqlite3
    con = sqlite3.connect('example.db')
 
-You can also supply the special name ``:memory:`` to create a database in RAM.
+Use the special path name ``:memory:`` to create a temporary database in RAM.
```

Comment:
I hear "use ... to ..." as short for "use ... in order to ...". This could be made explicit with "in order to", but I think it's clear that it's not a command. If it were a command, I would ask for it to be made explicit with "One must use..." etc

Reply:
I think it's partly the context of the rest of the documentation, which is written very much like a step-by-step recipe, that makes "use ... to" sound not quite right to me. But, just my opinion :)

Reply:
Personally, I don't think the existing wording sounds like a command, _but_ I like the suggested wording :) Thanks for helping out, both of you! Highly appreciated.